src/card.scss

```diff
@@ -364,4 +371,23 @@ $fd-card-default-body-background-color: var(--sapTile_Background) !default;
       padding: 1rem;
     }
   }
+
+  &__loader {
+    @include fd-reset();
+    @include fd-flex-center();
+
+    min-height: 5rem;
+  }
+
+  .#{$numeric-content}__scale-container {
+    @include fd-reset();
+
+    flex: auto;
```

Comment:
`flex: auto` is not working properly in IE11. Have you checked it?

Reply:
removed.